### ISSUE TITLE
collapsible group: make tabbable

### ISSUE THREAD

**Issue by paulschreiber** [usdigitalresponse/election-websites#85]
>Accordions should be tabbable with a focus state so they can be keyboard accessible.

**Comment by paulschreiber** [comment]
This is coming along. After talking to Josh:
• for pages that have two levels of accordion: first level is auto-expanded on open and not collapsible; second level gets tab index
• for pages that have one level of accordion: first level gets tab index
